assets/src/edit-story/app/prepublish/guidance/index.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Internal dependencies
+ */
+import * as generalGuidance from './general';
+import * as mediaGuidance from './media';
+import * as textGuidance from './text';
+
+export const story = [
+  generalGuidance.storyPagesCount,
+  generalGuidance.storyTitleLength,
+  textGuidance.storyTooLittleText,
+];
+export const page = [
+  mediaGuidance.videoElementSizeOnPage,
+  textGuidance.pageTooMuchText,
+];
+export const element = [
+  {
+    type: ['video', 'gif', 'image'],
```

Comment:
This works and is pretty easy to read, but for performance reasons since this is going to be running on _a lot_ of elements I think it would be better to avoid `includes`  and introduce a bit of duplication in returning of functions having to scan these arrays constantly and instead have this look more like

```
export const element = {
  all: [check, check],
  video: [check],
  gif: [check],
};
```

Reply:
I like this, thank you